drivers/soft_uart/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_SOFT_UART
+    bool "Software-implemented UART"
+    depends on HAS_PERIPH_GPIO
+    depends on HAS_PERIPH_GPIO_IRQ
+    depends on HAS_PERIPH_TIMER
+    depends on HAS_PERIPH_TIMER_PERIODIC
+    depends on TEST_KCONFIG
+    select MODULE_PERIPH_GPIO
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_TIMER
+    select MODULE_PERIPH_TIMER_PERIODIC
+    select HAS_PERIPH_UART # provides UART
```

Comment:
Unfortunately that doesn't work.
The soft UART mirrors the perih UART API, but uses a `soft_` prefix to avoid conflicts with the periph UART.
(The idea was to provide an additional UART to complement the HW UART)

Reply:
Yes unfortunately, we don't have a way to provide multiple backends for the periph API yet.

Reply:
Changed. I noticed it's also not the case for the mode configuration and soft SPI. Too bad :-(